README.md

```diff
@@ -114,9 +114,10 @@ Once installed the application is bound to `pykiso`, it can be called with the f
 ```bash
 Usage: pykiso [OPTIONS] [PATTERN]
 
-  Embedded Integration Test Framework - CLI Entry Point.
+  TAG Filters: Just add pairs of arguments to the pykiso call. Multiple values
+  for a key must be seperate via a comma.
 
-  PATTERN: overwrite the test filter pattern from the YAML file (optional)
+  For example: pykiso -c your_config.yaml branch dev,master variant delta
```

Comment:
I thought it should have been `--variant` ?

Reply:
fixed